### PR TITLE
Allow switching between GameTime and RealTime.

### PR DESCRIPTION
**Pull request by 0x0ade**
Still requires a setting to do that without recompiling the component.

### CODE REVIEW

**Comment by CryZe** [comment]
I do not approve. LiveSplit allows you to just use the Current Timing Method, which is more than enough. You can have a setting that overrides the Current Timing Method, but it shouldn't use the setting by default.

File: SplitsBetComponent.cs [hunk 1]
```diff
@@ -295,6 +297,10 @@ void State_OnReset(object sender, TimerPhase value)
             ResetSplitsBet();
         }
 
+        private TimeSpan? GetTime(Time segment) {
+            return UseGameTime ? segment.GameTime : segment.RealTime;
```
Comment (CryZe) [inline SplitsBetComponent.cs]: segment[timingMethod]
with:
TimingMethod timingMethod

**Comment by 0x0ade** [comment]
Good to know that's possible. I just did that because I've seen references to RealTime in the original code and thus added the possibility to switch between RealTime and GameTime instead of always using RealTime.

I still need to take a deeper look into LiveSplit.

**Comment by CryZe** [comment]
I guess you can modify GetTime to get the state too and then you can do something like this:

``` C#
TimingMethod timingMethod;
if (OverrideTimingMethod)
  timingMethod = OverriddenTimingMethod;
else
  timingMethod = State.CurrentTimingMethod;

return segment[timingMethod];
```

**Comment by 0x0ade** [comment]
Instead of using two properties, one being a boolean and another being the timing method, I'm using a nullable. Other than that, thanks for pointing out State.CurrentTimingMethod

**Comment by Gyoo** [comment]
State.CurrentTimingMethod might just be the ansyer to what I asked to Cryze this morning on Skype... If so, I'm dumb as a brick.

**Comment by Gyoo** [comment]
Shall I merge this or there's more to come ?

**Comment by 0x0ade** [comment]
If @CryZe can't find more issues, then I've got nothing to add.

**Pull request merged**